### DISCUSSION TITLE
What do the red dots mean on upper right hand corner, and what about a red box around the camera?

### DISCUSSION THREAD

**Discussion by CancunManny** [blakeblackshear/frigate#14257] [Ask A Question] [answered]
So I've tired to search and couldn't find the right answer.  Some posts state red dot means it is recording, however nothing is recorded.  Also I had an old Wyze cam that was giving me issues, and it appeared on the cameras view with a red box around it.  The feed would be iffy, guessing it was the cameras fault, but couldn't figure out what the red box meant
![image](https://github.com/user-attachments/assets/a9a5050e-fa59-4df4-bfe8-b7940ce0e9d4)

Those are the dots I am talking about.  And on that same screen/view is where I had a red box around one of the cameras.

**Comment by NickM-27** [comment, marked as answer]
It means the camera is currently detecting motion (not necessarily an object, just motion)

There will be a red border if an active object is detected

**Comment by CancunManny** [comment]
If I am understanding correctly red dot means there is motion, but I don't have anything configured to record on motion, only on detect, that is why I don't get any recordings.  If it does detect an active object, it will have red border and those are the ones where it does record then right?

**Reply by NickM-27** [reply]
Yes

**Comment by lpt2007** [comment]
Hi, I have red dot in right corner on all 4x cameras all the time. Why?

**Reply by hawkeye217** [reply]
If you have a constant red dot, it's likely that your cameras are constantly detecting motion. You should tune your motion in the Motion Tuner in Settings. 

https://docs.frigate.video/configuration/motion_detection

**Reply by lmorrison17** [reply]
If your cameras have a timestamp with changing seconds Frigate will see that as motion. You will need to put a motion mask over the cameras timestamp.

**Comment by Floriszz** [comment]
Thanks for clearing up the red dot a little bit.
I have a camera where I set the recording to false. Still I see the red dot. If I look in settings(Debug) I see it is detecting motions(red boxes). So motion detection keeps going, while you set recording to false. (already have motion masks around the camera timestamps of all my camera's👍)
